VideoAPI/VideoApi.IntegrationTests/Database/Queries/GetDistinctJudgeListByFirstNameQueryTests.cs

```diff
@@ -0,0 +1,96 @@
+using Faker;
+using FluentAssertions;
+using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Testing.Common.Helper.Builders.Domain;
+using VideoApi.DAL;
+using VideoApi.DAL.Queries;
+using VideoApi.Domain.Enums;
+
+namespace VideoApi.IntegrationTests.Database.Queries
+{
+    public class GetDistinctJudgeListByFirstNameQueryTests : DatabaseTestsBase
+    {
+        private GetDistinctJudgeListByFirstNameQueryHandler _handler;
+        private IList<Guid> _conferenceIds;
+
+        [SetUp]
+        public void Setup()
+        {
+            var context = new VideoApiDbContext(VideoBookingsDbContextOptions);
+            _handler = new GetDistinctJudgeListByFirstNameQueryHandler(context);
+            _conferenceIds = new List<Guid>();
+        }
+
+        [Test]
+        public async Task Should_get_a_list_of_judges()
+        {
+            for (int i = 0; i < 3; i++)
+            {
+                var newConference = new ConferenceBuilder(true)
+               .WithParticipant(UserRole.Representative, "Defendant")
+               .WithParticipant(UserRole.Judge, null)
+               .Build();
+                _conferenceIds.Add(newConference.Id);
+
+                await TestDataManager.SeedConference(newConference);
+            }
+
+            var judgelist = await _handler.Handle(new GetDistinctJudgeListByFirstNameQuery());
+
+            judgelist.Should().NotBeEmpty();
+        }
+
+        [Test]
+        public async Task Should_get_unique_list_of_judges()
+        {
+            //This is done to create an unique first name to avoid the test from failing because of any existing data
+            string judge1 = "Automation_" + Name.First() + Faker.RandomNumber.Next();
```

Comment:
Can you remove the comment and instead make it a function? CreateUniqueName()

Reply:
Good idea :)